backend/backend/urls.py

```diff
@@ -71,11 +71,11 @@
 
 	# Tournament views
 	path('tournaments/', TournamentListView.as_view()),
-	path('tournaments/<int:tournament_id>', TournamentDetailView.as_view()),
+	path('tournaments/<int:tournament_id>/', TournamentDetailView.as_view()),
 	path('tournaments/<int:tournament_id>/players/', TournamentPlayerListView.as_view()),
-	path('tournaments/<int:tournament_id>/players/<int:tournamentplayer_id>', TournamentPlayerDetailView.as_view()),
+	path('tournaments/<int:tournament_id>/players/<int:tournamentplayer_id>/', TournamentPlayerDetailView.as_view()),
 	path('tournaments/<int:tournament_id>/matches/', TournamentMatchListView.as_view()),
-	path('tournaments/<int:tournament_id>/matches/<int:tournament_match_id>', TournamentMatchDetailView.as_view()),
+	path('tournaments/<int:tournament_id>/matches/<int:tournament_match_id>/', TournamentMatchDetailView.as_view()),
```

Comment:
wasnt @sakarisalmi already fixing these in one PR?

Reply:
Shouldn't have, since we said, I'd do it...

Reply:
Alas he has... Ill  close this and add the append backslash later